### ISSUE TITLE
New Team member notification to be an Option. 

### ISSUE THREAD

**Issue by ianp5a** [GetTogetherComm/GetTogether#227]
When creating or editing a team a new checkbox _"Notify me of new members"_ is required.
Today an email notification is sent. For some this is not useful and is just spam. An option to disable it would be good.

**Comment by ianp5a** [comment]
I just read issue #188 **Notification (email) when user joins team / RSVPs for an event** regarding not having options. I can only see having a checkbox at create/edit of a Team as a small cost in order to reduce spam and keep those that need it happy.

**Comment by mhall119** [comment]
Do you want to continue receiving other notifications from GetTogether, just not when new members join your team?

**Comment by ianp5a** [comment]
Yes. This is specifically the New Member Notification. I don't know enough about other Team notifications.

**Comment by mhall119** [comment]
The reason I ask is because you can disable *all* notifications from GetTogether under your profile

**Comment by ianp5a** [comment]
I see. This must be separate. To clarify, the intention is to reduce spam, yet still know when something **important** occurs. If too many unwanted _"run of the mill"_ emails arrive, important ones will get missed. The setting should be granular and differentiate between reasons.

**Comment by craigmaloney** [comment]
I quite like getting a  notification of new users, but I also believe that what we have currently can be improved. Wondering if the subject line could have more information. I manage several teams so "New team members" for three teams all have the same subject. For Meetup the message I get is "New members in Michigan!/usr/group – UNIX/Linux users in Farmington Hills" which gives me more information before opening the email.

Might that help?

**Comment by mhall119** [comment]
Improving the email subject is a great idea, and I think should be a separate issue.

For team notifications, this shouldn't be a property of the team, in my opinion, since one admin might want to receive those emails and another might not. So we would need to make this property something separate from the team's properties.

**Comment by craigmaloney** [comment]
Created #230. Thanks!
